WordPress/Classes/ViewRelated/NUX/SiteCreationDomainsTableViewController.swift

```diff
@@ -207,6 +243,45 @@ extension SiteCreationDomainsTableViewController {
     }
 }
 
+// MARK: NoResultsViewController Extension
+
+private extension SiteCreationDomainsTableViewController {
+
+    func addNoResultsTo(cell: UITableViewCell) {
+        if noResultsViewController == nil {
+            instantiateNoResultsViewController()
+        }
+
+        guard let noResultsViewController = noResultsViewController else {
+            return
+        }
+
+        noResultsViewController.view.frame = cell.frame
+        cell.contentView.addSubview(noResultsViewController.view)
+
+        addChildViewController(noResultsViewController)
+        noResultsViewController.didMove(toParentViewController: self)
+    }
+
+    func removeNoResultsFromView() {
+        noSuggestions = false
```

Comment:
`removeNoResultsFromView` should also reset the value of `tableView.separatorStyle`

Reply:
It's set in the success & failure blocks of `service.getDomainSuggestions`. Adding `tableView.separatorStyle = .singleLine` in `removeNoResultsFromView` actually made no difference.